### ISSUE TITLE
debuginfo: Add metrics for `debuginfo` package

### ISSUE THREAD

**Issue by kakkoyun** [parca-dev/parca#1275]

**Comment by ksankeerth** [comment]
Hi @kakkoyun,
I'm new to Parca and learning eBPF and networking, Can I give it a try? Thanks

**Comment by metalmatze** [comment]
Yes, feel free to give it a try. 
If you need help join the Parca Discord and we can discuss there too.

**Comment by kakkoyun** [comment]
> Hi @kakkoyun, I'm new to Parca and learning eBPF and networking, Can I give it a try? Thanks

Yes, as metalmatze@ told, please go for it. It is about instrumenting the go APIs but definitely a good way to start contributing to Parca. You need to be familiar with how to instrument applications using @prometheus/client_golang.

**Comment by ksankeerth** [comment]
Thanks @kakkoyun  and @metalmatze. I have some familiarity with prometheus/client_golang. I'll give a try. If any help needed, I'll discuss with you.

**Comment by kakkoyun** [comment]
Similar attempt: #1276
